scripts/utilities/retry.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+
+/**
+ * Retry - Retry a function on failure
+ *
+ * @param {function} Run          - The function to retry on failure
+ * @param {string}   errorMessage - Additional information when the limit is reached
+ * @param {number}   attempts     - The total number of attempts
```

Comment:
I think this would be clearer if you renamed it to `currentAttempt`, what do you think?

Reply:
Nice one thanks!